ansible/playbook/tasks/update.yml

```diff
@@ -116,6 +117,11 @@
     - name: Check reread results
       debug: var=reread_result.stdout_lines
 
+    - name: Create working directory for Alertmanager
+      file: path=/srv/alertmanager state=directory owner=pmm group=pmm
```

Comment:
Mentioned in the other PR that it actually should be `/srv/alertmanager/data`

Reply:
@EvgeniyPatlan @hors @Sevlin Do you remember why we decided to do it there, not in RPM spec?